src/Tribe/Admin/Helpers.php

```diff
@@ -28,6 +33,41 @@ public static function instance() {
 		return self::$instance;
 	}
 
+	/**
+	 * Set context for tests
+	 *
+	 * @since TBD
+	 *
+	 * @return boolean
+	 */
+	public function doing_test( $doing_test = null ) {
```

Comment:
As much as I understand the idea behind this I'm not a fan of explicitly modifying code for testing purposes.
I've added comments later to clarify the concept and propose some different, possible approaches.

Reply:
Thanks for the feedback! I totally agree with every single point. It was one way to do it, but I definitely like yours better